todolist copy 3/main.js

```diff
@@ -1,111 +1,134 @@
-"use strict";
+'use strict'
 
-const addTaskButton = document.getElementById('addTask');
-const lineTaskInput = document.getElementById('taskInput');
-const toDoList = document.getElementById('tasklist');
-const clearing = document.getElementById('clearAll');
-const checkBoxCover = document.getElementById('clearAll');
+const addTaskButton = document.getElementById('addTask')
+const lineTaskInput = document.getElementById('taskInput')

```

Comment:
why lineTask? let's name it "input"

Reply:
so you've decided to use kebab case, then we need to rename "addTask", "taskInput" and "clearAll" to follow this naming convention too